assets/RFC003-AlphaBeta-AtomicSwap.puml

```diff
@@ -0,0 +1,142 @@
+' DIAGRAM #########################################
+' RFC003 atomic swap
+' #################################################
+@startuml
+' #################################################
+' SETTINGS: color settings of diagram
+' #################################################
+skinparam sequence {
+	BorderColor black
+	ArrowColor black
+	ActorBorderColor black
+	LifeLineBorderColor black
+	LifeLineBackgroundColor white
+	
+	ParticipantBorderColor black
+	ParticipantBackgroundColor white
+	ParticipantFontColor black
+	
+	ActorBackgroundColor white
+	ActorFontColor black
+}
+' #################################################
+' ACTORS
+' #################################################
+actor Alice
+participant "Alice\nComit-Node" as AC
+participant "Alpha\nLedger" as Alpha
+participant "Beta\nLedger" as Beta
+participant "Bob\nComit-Node" as BC
+actor Bob
+
+' allow parallelism
+!pragma teoz true
+
+' #################################################
+' DIAGRAM
+' #################################################
+Alice->AC: send swap request
+AC->AC: generate secret
+AC->AC: hash secret
+AC->BC: swap request 
+note left #white
+	swap request includes 
+	- hash of secret 
+	- expiries
+	- Alice's identities (if relevant)
+end note
+BC->Bob: new swap request
+Bob->BC: accept swap request
+BC->AC: accept swap request
+note right #white
+	swap accept includes 
+	- expiries
+	- Bob's identities (if relevant)
+end note
+BC->Alpha: monitor fund of HTLC on Alpha
```

Comment:
@D4nte @bonomat  in https://github.com/comit-network/RFCs/blob/master/RFC-005-SWAP-Basic-Bitcoin.md "Fund" is actually called "Deployment" - please upvote if I should change this.

It is somewhat related to the discussion about Fund/Redeem/Refund here: https://github.com/coblox/2019-genesis-workshop/issues/3

Reply:
A bit of an eternal debate. I'd be keen to change the RFC to "fund". IMHO, "Deployment" does not make sense in Bitcoin terms.

Reply:
Now that you have a generic diagram, you should have the "fund" action (optional) AND the deploy actions in the diagram. Check ERC20 basic htlc atomic swap RFC.

Reply:
I included the optional blocks with the "Fund" for tokens (e.g. ERC20) to give it a try how it looks.
I am not sure it helps the understanding of RCF003... maybe we could just obfuscate this detail by calling the step "deploy and fund" so it can include both? 
What do you think @bonomat ?

Reply:
I'm fine with that, wording is not that important to me :)